app/code/Magento/Eav/etc/db_schema_whitelist.json

```
@@ -307,7 +307,9 @@
         "constraint": {
             "PRIMARY": true,
             "EAV_ATTRIBUTE_LABEL_ATTRIBUTE_ID_EAV_ATTRIBUTE_ATTRIBUTE_ID": true,
-            "EAV_ATTRIBUTE_LABEL_STORE_ID_STORE_STORE_ID": true
+            "EAV_ATTRIBUTE_LABEL_STORE_ID_STORE_STORE_ID": true,
+            "EAV_ATTRIBUTE_LABEL_ATTRIBUTE_ID_STORE_ID_UNIQUE": true,
+            "EAV_ATTRIBUTE_LABEL_STORE_ID_ATTRIBUTE_ID": true
```

Please add the only needed key name:
```suggestion
            "EAV_ATTRIBUTE_LABEL_STORE_ID_STORE_STORE_ID": true,
            "EAV_ATTRIBUTE_LABEL_STORE_ID_ATTRIBUTE_ID": true
```

---

This one still not fixed, everything looks good